masq/src/commands/setup_command.rs

```diff
@@ -107,7 +111,28 @@ impl SetupCommand {
                 .expect("String comparison failed")
         });
         short_writeln!(stdout, "{:29} {:64} {}", "NAME", "VALUE", "STATUS");
+
+        let mut chain_and_data_dir = inner
```

Comment:
THis might be easier to understand if it were implemented as a pair of calls to `.find().map()`, perhaps passing a common closure to `map()`. Like so:
```
let to_triple = |p: Something| {(/* tuple guts from p */)};
let chain = inner.values.iter().find(|p| p.name.as_str() == "chain").map(to_triple).expect("Hey!");
let data_directory = inner.values.iter().find(|p| {p.name.as_str() == "data-directory").map(to_triple).expect("Hey!");
```